.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
from [eslint document](https://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories)

> In addition to any patterns in a `.eslintignore` file, ESLint always ignores files in `/node_modules/*` and `/bower_components/*`.

Reply:
removed